packages/rspack/tests/HotTestCases.template.js

```diff
@@ -64,35 +54,70 @@ export function describeCases(config: {
 									testName
 								);
 								rimraf.sync(outputDirectory);
-								// TODO: should remove it.
-								const changedFiles = path.join(
-									testDirectory,
-									"changed-file.js"
-								);
-
+								const recordsPath = path.join(outputDirectory, "records.json");
 								const fakeUpdateLoaderOptions = {
 									updateIndex: 0
 								};
 								const configPath = path.join(
 									testDirectory,
 									"webpack.config.js"
 								);
-								const options = getOptions(
-									configPath,
-									testDirectory,
-									outputDirectory,
-									fakeUpdateLoaderOptions,
-									config.target,
-									config.hot
+								let options = {};
+								if (fs.existsSync(configPath)) options = require(configPath);
+								if (typeof options === "function") {
+									options = options({ config });
+								}
+								if (!options.mode) options.mode = "development";
+								if (!options.devtool) options.devtool = false;
+								if (!options.context) options.context = testDirectory;
+								if (!options.entry) options.entry = "./index.js";
+								if (!options.output) options.output = {};
+								if (!options.output.path) options.output.path = outputDirectory;
+								if (!options.output.filename)
+									options.output.filename = "bundle.js";
+								// CHANGE: chunkFilename may cause errors in Rspack
```

Comment:
What's the specific error here?

Reply:
@h-a-n-a It is a bug about `runtime.js` chunk emit. I recorded it here: https://github.com/web-infra-dev/rspack/issues/5752